ui-v2/src/components/runs/runs-page.stories.tsx

```diff
@@ -48,6 +50,39 @@ const MOCK_FLOW_RUNS_TASK_COUNT = {
 	"4": randNumber({ min: 0, max: 5 }),
 };
 
+const MOCK_TASK_RUNS: TaskRunResponse[] = [
+	{
+		id: "task-0",
+		name: "my_task-0",
+		flow_run_id: "0",
+		task_key: "my_task",
+		state: { type: "COMPLETED", name: "Completed", id: "state-0" },
+		tags: [],
+		created: new Date().toISOString(),
+		updated: new Date().toISOString(),
+	},
+	{
+		id: "task-1",
+		name: "my_task-1",
+		flow_run_id: "1",
+		task_key: "my_task",
+		state: { type: "FAILED", name: "Failed", id: "state-1" },
+		tags: ["important"],
+		created: new Date().toISOString(),
+		updated: new Date().toISOString(),
+	},
+	{
+		id: "task-2",
+		name: "another_task-0",
+		flow_run_id: "2",
+		task_key: "another_task",
+		state: { type: "RUNNING", name: "Running", id: "state-2" },
+		tags: [],
+		created: new Date().toISOString(),
+		updated: new Date().toISOString(),
+	},
+];
+
```

Comment:
Use a mock from `ui-v2/src/mocks` or create one if the needed one doesn't already exist.